libs/api/windows_platform.cpp

```diff
@@ -104,13 +121,19 @@ int
 create_map_function(
     uint32_t type, uint32_t key_size, uint32_t value_size, uint32_t max_entries, ebpf_verifier_options_t options);
 
+void
+cache_map_file_descriptor(uint32_t type, uint32_t key_size, uint32_t value_size, int fd);
```

Comment:
Which file is this function in?  If in another file, move this to a .h file. If in this file, make it static.